Clarify limitation MCP trigger limitations in queue mode (#3084)
n8n-io/n8n-docs: docs/integrations/builtin/core-nodes/n8n-nodes-langchain.mcptrigger.md

## Limitations

> **MCP Trigger and Webhook Replicas**  

Reviewer: ```suggestion
### Configuring the MCP Server Trigger node with webhook replicas
```

> The MCP Trigger node relies on Server-Sent Events (SSE), which require the same server instance to handle persistent connections.

Reviewer: ```suggestion
The MCP Server Trigger node relies on Server-Sent Events (SSE), which require the same server instance to handle persistent connections. This can cause problems when running n8n in [queue mode](/hosting/scaling/queue-mode.md) depending on your [webhook processor](/hosting/scaling/queue-mode.md#webhook-processors) configuration:
```

> * If you run n8n in [queue mode](/hosting/scaling/queue-mode.md) with only a **single webhook replica**, MCP Trigger will work as expected.

Reviewer: ```suggestion
* If use queue mode with a **single webhook replica**, MCP Trigger works as expected.
```

> * If you run **multiple webhook replicas**, make sure that you route all `/mcp*` requests to a single, dedicated webhook replica.

Reviewer: ```suggestion
* If you run **multiple webhook replicas**, you need to route all `/mcp*` requests to a single, dedicated webhook replica. Create a separate replica set with one webhook container for MCP requests. Afterwards, update your ingress or load balancer configuration to direct all `/mcp*` traffic to that instance.
```

> * To achieve this, create a separate replica set with one webhook container for MCP requests, and update your ingress or load balancer configuration to direct all `/mcp*` traffic to that instance.

Reviewer: ```suggestion
```

> /// warning | MCP Trigger with Multiple Webhook Replicas

Reviewer: ```suggestion
/// warning | Running a MCP Server Trigger with multiple webhook replicas
```

> If you run MCP Trigger with multiple webhook replicas and don't route all `/mcp*` requests to a single, dedicated webhook replica, SSE connections will frequently break or fail to deliver events reliably.  

Reviewer: ```suggestion
If you run an MCP Server Trigger node with multiple webhook replicas and don't route all `/mcp*` requests to a single, dedicated webhook replica, your SSE connections will frequently break or fail to reliably deliver events.  
```
